TemplateApplication/Onboarding/AccountSetup/AccountSetup.swift

```diff
@@ -39,7 +43,12 @@ struct AccountSetup: View {
         )
             .onReceive(account.objectWillChange) {
                 if account.signedIn {
-                    moveToNextOnboardingStep()
+                    Task {
+                        moveToNextOnboardingStep(
+                            healthKitAuthorized: healthKitDataSource.authorized,
+                            notificationAuthorized: await scheduler.localNotificationAuthorization
```

Comment:
We don't really need these properties are arguments for the function, we can access the properties in the function.

Reply:
Good point!